pyuvsim/tests/profile_srcs.py

```diff
@@ -5,18 +5,14 @@
 from __future__ import absolute_import, division, print_function
```

Comment:
I don't remember what this script is used for, but I think it's out of date.

Reply:
Shall I remove it?

Reply:
Removed in b7e89d70fab47c00273dbfa95fa5f5c82f3af8b0